examples/commoncrawl/run_all_tests.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+collect () 
+{
+host=$1
+test=$2
+start=$3
+end=$4
+
+if  [  "$mlManageSimpleSsl" == "true" ]
+then
+   scheme="https"
+   args="--meters-ssl=true --meters-port=8002"
+else
+   scheme="http"
+   args=""
+fi
+
+
+
+curl  -s --anyauth --user $mlUsername:$mlPassword "$scheme://$host:8002/manage/v2?format=json"
+
+
+version=`curl  -s --anyauth --user $mlUsername:$mlPassword "$scheme://$host:8002/manage/v2?format=json" | jq '.["local-cluster-default"].version' `
+
+if  [  "$mlIsProvisionedEnvironment" == "true" ]
+then
+  hosts="" # we won't collect request logs from DHS
+  feature="dhs_websites"
+else
+  hosts=`curl  -s --anyauth --user $mlUsername:$mlPassword "$scheme://$host:8002/manage/v2/hosts?format=json" | jq  '.["host-default-list"]["list-items"]["list-item"][].nameref'`
+  feature="dhf_websites"
+
+fi
+
+version=${version//\"/}
+echo $version
+
+pear_host=rh7-intel64-80-dev-perf-1
+pear_user=builder
+pear_pass=builder
+
```

Comment:
We should not copy the credentials

Reply:
I'll msg you on Slack w/ some questions I have. This is PEAR credentials.